src/validations/css/prettycss.js

```diff
@@ -5,9 +5,34 @@ import endsWith from 'lodash/endsWith';
 
 const RADIAL_GRADIENT_EXPR =
   /^(?:(?:-(?:ms|moz|o|webkit)-)?radial-gradient|-webkit-gradient)/;
+
+const FILTER_VALUE_EXPR =
+    new RegExp([
+      'blur\\(|' +
+      'brightness\\(|' +
+      'contrast\\(|' +
+      'drop-shadow\\(|' +
+      'grayscale\\(|' +
+      'hue-rotate\\(|' +
+      'invert\\(|' +
+      'opacity\\(|' +
+      'saturate\\(' +
+      'sepia\\(|' +
+      'inherit|' +
+      'initial|' +
+      'unset'].join(''));
```

Comment:
Hmm, doesn’t seem like you’re getting much out of the `join` here—why not make a multi-element array without the pipes `['blur\\(', 'brightness\\(', …]` and then `join('|')`?